src/main/java/seedu/address/model/participation/Participation.java

```diff
@@ -34,6 +34,9 @@ public Person getStudent() {
     public Tutorial getTutorial() {
         return tutorial;
     }
+    public String getTutorialSubject() {
+        return this.tutorial.getSubject();
```

Comment:
Could we use tutorial instead of this.tutorial?

Reply:
Not sure if i remembered correctly but "this" should be used only if theres another conflicting variable name in the method. Need to check on this again.

Reply:
okayy i will change this too